exporter/pwd_file_test.go

```diff
@@ -0,0 +1,28 @@
+package exporter
+
+import (
+	"log"
+	"testing"
+)
+
+func TestLoadPwdFile(t *testing.T) {
+	confFile := "../contrib/sample-pwd-file.json"
+	passwordMap, err := LoadPwdFile(confFile)
+	if err != nil {
+		t.Fatalf("Test Failed, error: %v", err)
+	}
+
+	redisHostList := []string{
```

Comment:
Similar to my other comment, see [here](https://github.com/oliver006/redis_exporter/blob/master/exporter/exporter_test.go#L1544-L1563) what a test layout looks like.
Define a struct with some inputs and expected outputs, then iterate over it and fail if outputs don't match the expected results.

Reply:
I think here only needs to test whether the contents of the password file can be read correctly.